torch/_inductor/ir.py

```diff
@@ -7210,15 +7212,15 @@ def codegen(self, wrapper) -> None:  # type: ignore[no-untyped-def]
 @ir_dataclass(frozen=False)
 class Conditional(ExternKernel):
     predicate: Optional[IRNode] = None
-    operands: Optional[list[TensorBox]] = None
+    operands: Optional[List[Union[TensorBox, ShapeAsConstantBuffer]]] = None
```

Comment:
List->list is a recent update, https://github.com/pytorch/pytorch/pull/145137. Please don't undo it.